test/format/format_config.c

```diff
@@ -1386,6 +1388,43 @@ config_statistics(void)
     }
 }
 
+/*
+ * config_tiered_storage --
+ *     Tiered storage configuration.
+ */
+static void
+config_tiered_storage(void)
+{
+    const char *storage_source;
+
+    storage_source = GVS(TIERED_STORAGE_STORAGE_SOURCE);
+
+    g.tiered_storage = (strcmp(storage_source, "off") != 0 && strcmp(storage_source, "none") != 0);
+    if (g.tiered_storage) {
+        /* Tiered storage requires timestamps. */
+        config_off(NULL, "transaction.implicit");
+        config_single(NULL, "transaction.timestamps=on", true);
+
+        /* If we are flushing, we need a checkpoint thread. */
+        if (GV(TIERED_STORAGE_FLUSH_FREQUENCY) > 0)
+            config_single(NULL, "checkpoint=on", false);
+
+        /* FIXME-WT-8727: Salvage and verify are not yet supported for tiered storage. */
+        config_off(NULL, "ops.salvage");
+        config_off(NULL, "ops.verify");
+
+        /* FIXME-PM-2532: Backup is not yet supported for tiered tables. */
+        config_off(NULL, "backup");
+        config_off(NULL, "backup.incremental");
+
+        /* FIXME-PM-2538: Compact is not yet supported for tiered tables. */
+        config_off(NULL, "ops.compaction");
```

Comment:
Do we need to assert or otherwise check that `tiered_storage.flush_frequency` is > 0 here, if the `else` clause below turns it off with 0?